documentation/src/main/asciidoc/topics/rest_examples/get_v2_all_details.adoc

```diff
@@ -0,0 +1 @@
+GET /rest/v2/caches/{name}?action=all
```

Comment:
There is not need for the ```action``` here, I'd just keep ```GET /rest/v2/caches/{name}``` as it's not being used for any resource currently

Reply:
ok!